sdk/tests/wallet/wallet_storage.rs

```diff
@@ -85,6 +86,55 @@ async fn check_existing_db() -> Result<()> {
     tear_down(storage_path)
 }
 
+// Db created with wallet.rs commit 2dd9974c1bc05c2b0b7d6f0ee100deb2da60d071
+#[cfg(all(feature = "ledger_nano", feature = "rocksdb"))]
+#[tokio::test]
+async fn check_existing_db_1() -> Result<()> {
+    let storage_path = "check_existing_1_db_test";
+    setup(storage_path)?;
+    // Copy db so the original doesn't get modified
+    copy_folder("./tests/wallet/fixtures/check_existing_1_db_test", storage_path).unwrap();
+
+    let wallet = Wallet::builder().with_storage_path(storage_path).finish().await?;
+
+    matches!(
+        *wallet.get_secret_manager().read().await,
+        iota_sdk::client::secret::SecretManager::LedgerNano(_)
+    );
```

Comment:
this does not make the test fail if it doesn't match

Reply:
Good catch, should probably have an assert on it

Reply:
wow 3 approvals, I must be missing something :trollface:

Reply:
haha, okay

Reply:
Pinging people works

Reply:
Done

Reply:
Thanks

Reply:
I'll allow myself to resolve this